app/views/keywords/index.html.erb

```diff
@@ -1,3 +1,4 @@
-<div class="container">
+<div class="container screen-keywords">
```

Comment:
Lets use singular class `screen-keyword`

Reply:
Ok, then I've renamed the javascript screen class/file too, to keep consistency ;-)